### PR TITLE
Update svgbob to build on stable again and fix maketitle short author for non-beamer

### PR DESCRIPTION
**Pull request by oberien**

### CODE REVIEW

File: src/backend/latex/preamble.rs [hunk 1]
```diff
@@ -121,8 +133,10 @@ pub fn write_maketitle_info(cfg: &Config, out: &mut impl Write) -> Result<()> {
     }
     if cfg.author.is_some() || cfg.supervisor.is_some() || cfg.advisor.is_some() {
         write!(out, "\\author")?;
-        if let Some(author) = &cfg.author {
-            write!(out, "[{}]", author)?;
+        if short_author == ShortAuthor::Yes {
+            if let Some(author) = &cfg.author {
+                write!(out, "[{}]", author)?;
```
Comment (197g) [inline src/backend/latex/preamble.rs:141]: What's the need for writing the exact same author again? Shouldn't this be complemented by another config option for allowing some customization of the short style and defaulting to the original option?

**Review by 197g — CHANGES_REQUESTED** [review]

File: src/backend/latex/preamble.rs (hunk 1 above)
Comment (oberien) [reply src/backend/latex/preamble.rs:141]: If I don't provide a short author, IIRC no author is shown in the footer of slides at all.

Comment (oberien) [reply src/backend/latex/preamble.rs:141]: I seem to remember wrongly…

Comment (oberien) [reply src/backend/latex/preamble.rs:141]: I opened #125 for it.

**Pull request merged**